xml/libvirt_managing.xml

```diff
@@ -885,8 +887,8 @@ Metadata:       yes
    <sect3 xml:id="sec.libvirt.managing.snapshots.virsh.delete">
     <title>Deleting a Snapshot</title>
     <para>
-     To delete a snapshot of a &vmguest; and restore the disk space it
-     occupies, use the <literal>snapshot-delete</literal> command:
+     External snapshots could be deleted here. To delete an internal snapshot of a &vmguest;
```

Comment:
After looking at bsc#1125660 and the article that was referenced there, I think this sentence should be:
'External snapshots cannot be deleted with ```<command>virsh</command>```.'

Reply:
Really appreciate the review, I changed the parts you mentioned, please have a look and let me know if have further questions.